docker/docker-compose.yml

```diff
@@ -100,6 +101,11 @@ volumes:
   bitcoin:
     driver: local
 
+  # lnd volume is used for persisting lnd application data and chain state
+  # during container lifecycle
```

Comment:
nit: missing full stop at end of sentence

Reply:
:+1: